### DISCUSSION TITLE
CAN SOMEONE SEND ME 32BIT DBEAVER FOR WINDOWS?

### DISCUSSION THREAD

**Discussion by tia000000000** [dbeaver/dbeaver#14030] [General]
Hi everyone! I tried installing DBeaver but for some reason it says it doesn't match my Windows. I downloaded many of the archived versions but unfortunately even those whom I know are 32bit, they were still not working. I have Windows 7 Ultimate. I wonder if someone knows another ways of downloading and finding 32bit Beavers. Thanks in advance!

**Comment by ShadelessFox** [comment]
Unfortunately, DBeaver requires Windows x64.

Why would you use 32-bit operating system nowadays? Your CPU should support AMD64 architecture.

**Comment by p91paul** [comment]
On https://dbeaver.io/download/ there is a "Previous versions archive" at the bottom of the links. From what I see, 6.0.0 is the latest version which provides an x86 download.
